### PR TITLE
moved autoinst_url() to the backend to get different behaviour between s390x and other archs

### PR DESCRIPTION
**Pull request by Soulofdestiny**
please don't accept until sr#318154 was deployed

### CODE REVIEW

File: testapi.pm [hunk 1]
```diff
@@ -484,10 +484,10 @@ returns the base URL to contact the local os-autoinst service
...
 =cut
 
-sub autoinst_url() {
-    # move to backend?
-    return "http://10.0.2.2:" . (get_var("QEMUPORT") + 1);
-}
+#This was moved to backend
+#sub autoinst_url() {
+#    return "http://10.0.2.2:" . (get_var("QEMUPORT") + 1);
+#}
```
Comment (aaannz) [inline testapi.pm]: I don't think keeping comments is necessary

File: backend/s390x.pm [hunk 2]
```diff
@@ -18,6 +18,8 @@ use backend::VNC;
 
 use testapi qw(get_var check_var set_var);
 
+use Net::Address::IP::Local;
```
Comment (aaannz) [inline backend/s390x.pm]: You might want to update DEPENDENCIES.txt

**Comment by coolo** [comment]
did you test this actually?

File: backend/s390x.pm (hunk 2 above)
Comment (lnussel) [inline backend/s390x.pm]: not properly packaged anyways. Note that this module tries to connect to a server in the internet to find out it's own address.

**Pull request closed without merging**